verifiers/utils/dsse.go

```diff
@@ -42,13 +43,17 @@ func PayloadFromEnvelope(env *dsselib.Envelope) ([]byte, error) {
 	return payload, nil
 }
 
+// StatementFromBytes parses the provided byte slice as a JSON payload and returns an intoto.Statement.
+// Ideally, we use the "V1" Statement in https://pkg.go.dev/github.com/in-toto/attestation/go/v1#pkg-constants,
+// but it parses json fields in snake case, while the official spec uses camel case
+// https://github.com/in-toto/attestation/blob/v1.0/spec/v1.0/statement.md.
 func StatementFromBytes(payload []byte) (*intoto.Statement, error) {
 	var statement intoto.Statement
 	if err := json.Unmarshal(payload, &statement); err != nil {
 		return nil, fmt.Errorf("%w: %w", serrors.ErrorInvalidDssePayload, err)
 	}
 
-	if statement.Type != intoto.StatementInTotoV01 {
+	if statement.Type != intoto.StatementInTotoV01 && statement.Type != intotoAttestations.StatementTypeUri {
```

Comment:
do you intend to be a `&&` or an `||`?

Reply:
Definitely a `&&`, so we can say "if it's neither a V01 or V10, then error."But we can change it to a more proper NOR or use a map like we've done in other parts of the code.

Reply:
my bad, mis-read the code. Thanks for the exaplanation